src/omni_epd/displays/mock_display.py

```diff
@@ -55,13 +55,16 @@ def prepare(self):
         self.logger.info(f"preparing {self.__str__()}")
 
     def _display(self, image):
-        self.logger.info(f"{self.__str__()} writing image to {self.output_file}")
+        if(self._getboolean_device_option('write_file', True)):
+            self.logger.info(f"{self.__str__()} writing image to {self.output_file}")
 
-        if(image.mode == 'P'):
-            # can't write P mode images
-            image = image.convert('RGB')
+            if(image.mode == 'P'):
+                # can't write P mode images
```

Comment:
You can if you use PNG or GIF.

Reply:
Nice, I checked the Pillow docs but it didn't mention that, just threw the error.